qa/standalone/osd/osd-rep-recov-eio.sh

```diff
@@ -44,9 +44,10 @@ function run() {
 function setup_osds() {
     local count=$1
     shift
+    local type=$1
```

Comment:
shall we default to `bluestore` here? like
```sh
local type =${2:-_bluestore}
```

Reply:
@tchaikov No, there is no longer a run_osd_bluestore().  Normally, we use run_osd() which currently defaults to bluestone.  There is now a run_osd_filestore() as a special type.